src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
Yeah, that sounds like a breaking change. I tried to keep it as conflict-free as possible.